src/Octokit.Webhooks/WebhookEvent.cs

```diff
@@ -17,4 +17,10 @@ public abstract record WebhookEvent
 
     [JsonPropertyName("sender")]
     public User? Sender { get; init; }
+
+    /// <summary>
+    /// Gets any additional properties that were not mapped to a strongly-typed property on the event.
+    /// </summary>
+    [JsonExtensionData]
+    public IDictionary<string, dynamic>? AdditionalProperties { get; init; }
```

Comment:
Would `JsonElement` work? If it does that maybe feels a bit safer to me than going down to dynamic as it would encourage people to be more type safe rummaging around in it if they need to.

I like the idea of an escape-hatch though.

Reply:
Yeah, good idea. I'll change this to `IDictionary<string, JsonElement>?` instead.